src/libsystemd/libsystemd.sym

```diff
@@ -576,4 +576,64 @@ global:
         sd_bus_message_readv;
         sd_bus_set_method_call_timeout;
         sd_bus_get_method_call_timeout;
+
+        sd_device_ref;
+        sd_device_unref;
+
+        sd_device_new_from_syspath;
```

Comment:
We have never add so many blank lines...

Reply:
Yeah. But I think it makes the whole thing a bit more readable... I kept the same structure as in the .h file.